### ISSUE TITLE
Converting text to speech

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#56]
## User story
1. As a user
2. I want / need to be able to report my problem / issue, request service in speech.
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
